### PR TITLE
removed support for PDB on Windows...

### PR DESCRIPTION
**Pull request by richardjgowers**
Starts #2125

Changes made in this Pull Request:
 - makes PDB reading fail on Windows (for now)

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/core/test_atomselections.py [hunk 1]
```diff
@@ -344,13 +344,10 @@ def test_type(self, universe):
...
 
 
-@pytest.mark.xfail(os.name == 'nt',
-                   strict=True,
-                   reason="Not supported on Windows yet.")
 class TestSelectionsNAMD(object):
     @pytest.fixture()
     def universe(self):
-        return MDAnalysis.Universe(PSF_NAMD, PDB_NAMD)
+        return MDAnalysis.Universe(PSF_NAMD)
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/core/test_atomselections.py]: These selections only use topology, so we only need a PSF-universe

**Comment by richardjgowers** [comment]
I think once/if this merges, then everything "accessible" from Windows should work...

**Comment by orbeckst** [comment]
This PR can only be an initial step towards getting this to work. We cannot ship a WIndows build that cannot work with PDB files.

**Comment by richardjgowers** [comment]
@orbeckst ngl just uses a single PDB to grab topology, (ie it serialises a Universe via a PDB save/load) then coords are slurped straight from our Reader, so a multiframe PDB traj wouldn't work

I'm playing with a windows VM now... maybe the PDB thing is an easy fix....

**Pull request closed without merging**